package.json

```diff
@@ -29,7 +29,7 @@
   "license": "MIT",
   "homepage": "https://github.com/companieshouse/overseas-entities-web#readme",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.174",
+    "@companieshouse/api-sdk-node": "^2.0.176",
```

Comment:
You need to also include the changed package-lock.json file now that version is updated.

Reply:
Included package lock